tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -3569,7 +3603,16 @@ def info(self):
     # TODO: Move code logic to learner/trainer
     # TODO: Add fit() arg to perform this automatically at end of training
     # TODO: Consider adding cutoff arguments such as top-k models
-    def fit_weighted_ensemble(self, base_models: list = None, name_suffix="Best", expand_pareto_frontier=False, time_limit=None, refit_full=False):
+    def fit_weighted_ensemble(
+        self,
+        base_models: list = None,
+        name_suffix="Best",
```

Comment:
nitpick on consistency: there are arguments with / without type, and with / without space around =

Reply:
Good point. I was initially not planning to touch the original code when adding the additional arguments beyond satisfying the linter, but I have went and updated to add type hints.

Regarding the spaces, this is because when you do type hints, the correct linting is to add spaces, but without type hint, you should not have spaces.